packages/sdk/src/controllers/ViewModeController.ts

```diff
@@ -0,0 +1,30 @@
+import { EditorAPI } from '../types/CommonTypes';
+import { ViewMode } from '../types/ViewModeTypes';
+import { getEditorResponseData } from '../utils/EditorResponseData';
+
+/**
+ * The ViewModeController is responsible for switching view modes.
+ */
+export class ViewModeController {
+    /**
+     * @ignore
+     */
+    #editorAPI: EditorAPI;
+
+    /**
+     * @ignore
+     */
+    constructor(editorAPI: EditorAPI) {
+        this.#editorAPI = editorAPI;
+    }
+
+    /**
+     * This method sets a new view mode
+     * @param viewMode view mode of the editor
+     * @returns
+     */
+    set = async (viewMode: ViewMode) => {
```

Comment:
```suggestion
    setViewMode = async (viewMode: ViewMode) => {
```
To be inline with other controllers. You don't know if we're going to extend this

Reply:
I wanted to ask on Studs as well because indeed `ToolController` is the same as you showed.

What is "_extending_" in this context? Won't `viewMode.set` always have the same intent?

Reply:
you are right, but question is if the controller isn't too specific?